scabha/evaluator.py

```diff
@@ -176,6 +176,11 @@ def make_list(*x):
             return list(x)
         return self.evaluate_generic_callable(evaluator, "LIST", make_list, args)
 
+    def LEN(self, evaluator, args):
+        def make_len(x):
+            return len(x)
+        return self.evaluate_generic_callable(evaluator, "LEN", make_len, args)
```

Comment:
@tmolteno please add min_args=1, max_args=1 to this call, so that it throws an error message if things are otherwise.

Reply:
@tmolteno reminder of requested change here.